variables.tf

```diff
@@ -142,6 +142,12 @@ variable "public_ip" {
   description = "Assign a public ip to the service"
 }
 
+variable "readonlyRootFilesystem" {
```

Comment:
Terraform variables should be like this: readonly_root_filesystem

Reply:
👍